dev_guide/builds.adoc

```diff
@@ -11,11 +11,78 @@ toc::[]
 
 == Overview
 A link:../architecture/core_objects/builds.html[build] is a process of creating
-runnable images to be used on OpenShift. There are three types of builds:
+runnable images to be used on OpenShift. There are three build strategies:
 
-- Docker build
-- STI build
-- custom build
+- link:../architecture/core_objects/builds.html#source-build[Source-To-Image]
+- link:../architecture/core_objects/builds.html#docker-build[Docker]
+- link:../architecture/core_objects/builds.html#custom-build[Custom]
+
+== Defining a buildConfig
+
+A `*buildConfig*` describes a single build definition and a
+set of link:#triggers[triggers] for when a new build should be created.
+
+A buildConfig is a REST object, which can be used in a POST to the API server to
+create a new instance. The following configuration results in a new build
+every time a Docker image tag or the source code changes:
+
+[source.json]
+----
+{
+  "kind": "BuildConfig",
+    "apiVersion": "v1",
+    "metadata": {
+      "name": "ruby-sample-build <1>",
+    },
+  "spec": {
+    "triggers": [ <2>
```

Comment:
I would change the order of the fields here to have triggers presented last. When reading the description below, right in the second point we talk about triggers and have to make cross-reference to `source` and `strategy` which were not explained just yet.